tools/dbutility/README.md

```diff
@@ -0,0 +1,19 @@
+### dump_db_stats utility
+
+This utility helps in analysing the hyperledger db contents; particularly, this utility prints
+- the key-values which are over 1 MB (see const MaxValueSize, below),
+- Further details about the key-values (e.g., number of transactions and over-sized transactions in the case of blockchain column family)
+- LiveFilesMetaData about rocksdb .sst files
+- Certain properties about rocksdb such as num-live-versions and cfstats (see 'struct Properties' at https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h)
+
+This utility can be run only on a off-line copy of the rocksdb i.e, the rocksdb instance that is not being used by a hyperledger peer currently.
+
+Though, this utility does not modify the db contents in any manner but rocksdb library may run background activities
```

Comment:
"contents in any manner but rocksdb library" -> "contents in any manner, the rocksdb library"